tests/utils.py

```diff
@@ -206,31 +206,16 @@ def __init__(self, user):
 
 def unique(factory):
     """
-<<<<<<< HEAD
     Turn a factory function into one that guarantees unique return values.
-=======
-    Turns a factory function into a new factory function that guarantees unique return
-    values. Note this uses regular item equivalence to check uniqueness, so this may not
-    behave as expected with factories with complex return values.
-
-    Example use:
-    unique_name_factory = unique(fake.name)
-    unique_name = unique_name_factory()
->>>>>>> upstream/hotfix/25.18.1
```

Comment:
Nitpicking: I think I commented on a different one a similar thing. Is there any value in the original docstring that worth mention in our new ones. The old one seems to provide more context but maybe it's no longer relevant in NR?